packages/mobile/src/send/SendAmount.tsx

```diff
@@ -241,6 +233,10 @@ export class SendAmount extends React.PureComponent<Props, State> {
     }
 
     const confirmationInput = this.getConfirmationInput()
+    if (!confirmationInput) {
+      return
```

Comment:
what could cause a failure case here? should we throw?

Reply:
Good catch I don't need this anymore. I had to add this when `fee` was part of `ConfirmationInput`. But I've now removed it.